tests/acl/templates/acltb_test_stress_50_plus_rules.json

```diff
@@ -0,0 +1,626 @@
+{
```

Comment:
Why we set the number of rules to 50? For a stress test, 50 seems not enough.

Reply:
How many rules can be considered sufficient for a stress test such as this? There are 74 rules which is more than 50 as specified in https://github.com/sonic-net/sonic-mgmt/issues/3595.

Reply:
I suggest using different scales for different platforms. For most modern platforms, the scale can be over 512. And for some legacy platform, it could be 256. We can define a mapping from platform to the number of rules. But that requires some changes in code structure. For example, the json file needs to be generated dynamically.